docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,170 @@
+
+The workbook object is the entry point for your add-in to interact with Excel. It contains a [WorksheetCollection](/javascript/api/excel/excel.worksheetcollection) to add worksheets, navigate among them, and assign handlers to worksheet events. The article [Work with Worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md) describes how to access and edit worksheets.
+
+## Get active cell and selected range
+
+Workbooks maintain collections of worksheets, tables, PivotTables, and other data types through which Excel data is accessed and changed. They also have application-level information (more on that later). There are a pair of methods to directly get ranges the Excel user has just activated. The first is `getActiveCell()`. This method gets the active cell from the workbook as a [Range object](/javascript/api/excel/excel.range). The following example shows a call to `getActiveCell()`, followed by the cell's address being printed to the console.
+
+
+```js
+Excel.run(function (context) {
+    var activeCell = context.workbook.getActiveCell();
+	activeCell.load("address");
+
+    return context.sync().then(function () {
+		console.log("The active cell is " + activeCell.address);
+	});
+}).catch(errorHandlerFunction);
+```
+
+The second range access method is `getSelectedRange()`. This method returns the range currently selected by either a user or an add-in. The following example shows a call to `getSelectedRange()` that then sets the range's fill color to yellow.
+
+```js
+Excel.run(function(context) {
+	var range = context.workbook.getSelectedRange();
+	range.format.fill.color = "yellow";
+	return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+## Create workbook
+
+Your add-in can create a new workbook, separate from the Excel instance in which the add-in is currently running. The Excel object has the `createWorkbook` method for this purpose. When this method is called, the new workbook is immediately opened and displayed. This happens in a new Excel instance. Your add-in remains open and running with the previous workbook.
+
+```js
+Excel.createWorkbook();
+```
+
+`createWorkbook` takes in an optional string. This string is an .xlsx file in base64 encoding. Assuming the string argument is a valid .xlsx file, the resulting workbook will be a copy of that file. You can get your add-in’s current workbook as a base64-encoded string by using [file slicing](/javascript/api/office/office.document#getfileasync-filetype--options--callback-). The [FileReader](https://developer.mozilla.org/docs/Web/API/FileReader) class can be used to convert a file into the required base64-encoded string, as demonstrated in the following example. 
```

Comment:
- Consider rephrasing first sentence -> "The `createWorkbook` method accepts an optional string parameter."

- Consider rephrasing second sentence -> "This parameter is a base64-encoded string that represents the .xlsx file that the ` createWorkbook` method will use as a basis for creating the new workbook."

- Consider adding a line break after sentence # 3 and then rephrasing the start of next sentence to describe the scenario that the following guidance and code example addresses -> "To create a new workbook that's a copy of the current workbook,..." (or something similar...as it's currently written, it's not clear what scenario the example addresses)